src/test/java/gov/healthit/chpl/aqa/pageObjects/ListingDetailsPage.java

```diff
@@ -327,6 +327,15 @@ public static WebElement surveillanceActivitiesAccordion(final WebDriver driver)
         return driver.findElement(By.id("details-surveillance-activities"));
     }
 
+    /**
+     * Returns Targeted User data from additional Info panel.
+     * @param driver WebDriver
+     * @return the text element
+     */
+    public static WebElement targetedUser(final WebDriver driver) {
+        return driver.findElement(By.xpath("//*[@id=\"panel-additional-information\"]/span[2]/ul/li"));
```

Comment:
I updated the website to have some IDs for these areas. They're in this PR: https://github.com/chpladmin/chpl-website/pull/405

The reason is that the position of all of these items is dependent on which other items exist in the Listing, so you can't just count the 2nd or 3rd ones. Recommend changing both this and the meaningfulUse page object definition

Reply:
Awesome. Thanks! I'll update locators once PR is merged.